.gitignore

```diff
@@ -111,6 +111,10 @@ Server/cuberite_api.lua
 Server/official_undocumented.lua
 Server/NewlyUndocumented.lua
 Server/.luacheckrc
+compile_commands.json
 
 # compile.sh folder
 build-cuberite
+
+# clang-tidy
+tidy-build
```

Comment:
If you don't plan to put `run-clang-tidy.py` into this repo, then add it to the list of ignored files.

Reply:
I'll add it, although not really necessary since you would just add the file to your PATH (after #4225 `run-clang-tidy.py` should be in PATH)

Reply:
If you intent, that everyone should have the file in the path, you should remove the `../run-clang-tidy.sh` check and just add the cuberite path to the `PATH` variable in CircleCI 1.